spacecmd/src/spacecmd/group.py

```diff
@@ -550,14 +560,12 @@ def do_group_addconfigchannels(self, args):
         return 1
 
     arg_parser = get_argument_parser()
-    (args, options) = parse_command_arguments(args, arg_parser)
+    (args,) = parse_command_arguments(args, arg_parser)
```

Comment:
This can't work, at least not if the previous version worked:
```python
In [1]: def returns_tuple_of_two():
   ...:     return (100, "100")
   ...: 

In [2]: (args,) = returns_tuple_of_two()
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-2-627d73f70b4f> in <module>
----> 1 (args,) = returns_tuple_of_two()

ValueError: too many values to unpack (expected 1)
```

The reason is that `(args, )` is a tuple of length 1:
```python
In [4]: len((args,))
Out[4]: 1
```

I assume the change is done because `options` was never used. In that case, I would use `_`. It's the go-to placeholder variable name for variables that are not accessed. I don't know if `pylint` will still complain, haven't tried it yet.

```python
In [3]: (args,_) = returns_tuple_of_two()
# no error, values are saved in args and _
```

Reply:
Thank you for giving the explanation and examples!